code/__DEFINES/dcs/signals.dm

```diff
@@ -264,6 +264,12 @@
 #define COMSIG_MOVABLE_BUCKLE "buckle"
 ///from base of atom/movable/unbuckle_mob(): (mob, force)
 #define COMSIG_MOVABLE_UNBUCKLE "unbuckle"
+///from /atom/movable/proc/buckle_mob, used by the ridable element to see if someone can successfully ride the target
+#define COMSIG_MOVABLE_TRY_MOUNTING "try_mounting"
+	#define MOUNTING_HALT_BUCKLE	(1<<0)
```

Comment:
```suggestion
	#define BUCKLE_MOB_CANCEL	(1<<0)
```
or something like that, let's not keep multiple names for the same procs and events